### ISSUE TITLE
Possible bug involving INCLIST and trailing forward-slashes

### ISSUE THREAD

**Issue by kdambrosio** [zertrin/duplicity-backup#113]
duplicity-backup.conf.example contains the following example for backing up a single location:

    INCLIST=( "/home/foobar_user_name/Documents/" )

But in practice, leaving the trailing forward-slash on the folder you want to back up causes *only the folders* to be backed up, and not their contents. Duplicity-backup.sh will only list the folders inside **Documents/** when the **-l** flag is used, and none of the actual files can be restored. I noticed this issue when the backup of my Pictures folder was not even a megabyte large. 

I might be using an old version of duplicity-backup.sh, but the script doesn't seem to output a version number so I'm not sure how to check. Removing the trailing forward-slash solves the problem. Is this a bug?

**Comment by arichiardi** [comment]
I think this can be related with the way [`duplicity` handles file selection](http://duplicity.nongnu.org/duplicity.1.html#sect9)

**Comment by zertrin** [comment]
Yeah, I'm not sure myself, I find the explanations about file selection quite hard to understand in the official documentation of duplicity. And don't see anything specifically speaking about trailing slashes in paths.

In any case, this issue is probably not caused by our script itself, since the paths given in `INCLIST` and `EXCLIST` are passed as is to duplicity.

I guess we should remove the final slash from the example. Not sure I can do more, but I'm open to suggestions.

**Comment by arichiardi** [comment]
I can confirm the problem and the workaround, just add the option to static ones (with newer `duplicity`):

`STATIC_OPTIONS="--full-if-older-than 30D --exclude-filelist ~/.backup/HomeBackup.duplicity"`

**Comment by pcoultha** [comment]
This Duplicity behavior is being tracked and discussed here:
https://bugs.launchpad.net/duplicity/+bug/1624725

It's also worth looking at this earlier request to re-introduce this behavior here:
https://bugs.launchpad.net/duplicity/+bug/1479545

**Comment by zertrin** [comment]
Thanks for the heads-up!
